src/common/telemetry/src/logging.rs

```diff
@@ -420,6 +426,42 @@ pub fn init_global_logging(
     guards
 }
 
+fn build_otlp_exporter(opts: &LoggingOptions) -> SpanExporterBuilder {
+    let protocol = opts
+        .otlp_export_protocol
+        .clone()
+        .unwrap_or(OtlpExportProtocol::Http);
```

Comment:
Is this a breaking change since the default protocol and port are changed to HTTP instead of gRPC